real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/RobobuggyMainFile.java

```diff
@@ -40,32 +41,18 @@ public static void main(String[] args) {
         System.out.println(ports);
         Robot.getInstance();
         Gui.getInstance();
+
      	
     	if (RobobuggyConfigFile.DATA_PLAY_BACK) {
     		//Play back mode enabled
-    		new SensorPlayer("logs/2016-02-12-21-52-18/sensors_2016-02-12-21-52-18.txt", 1);
+    		new SensorPlayer("logs/spring1data/2016-02-20-06-50-45/sensors_2016-02-20-06-50-45.txt", 1);
+
         }
-        else {
+        
         	//Play back disabled, create robot
         	Robot.getInstance().startNodes();
```

Comment:
I disagree, we may want to do play back which sends commands to the actual buggy.  Hardware in the loop testing with logged sensor values.   This is actually how I debugged and fixed the gps driving system that we currently use.